### PR TITLE
Fixes #23318. Apply postgres DB options to dbshell

### PR DESCRIPTION
**Pull request by dulmandakh**
When I use *django.db.backends.postgresql*, and set OPTIONS like below, it would apply to psycopg2 connections, but not dbshell. This PR fixes the inconsistency, thus improve DX.

```python
DATABASES = {
    "default": {
        "ENGINE": "django.db.backends.postgresql",
        "NAME": "db",
        "HOST": "127.0.0.1",
        "OPTIONS": {"options": "-c default_text_search_config=simple"}
    }
}
```

Without this patch entering below command in dbshell would show pg_catalog.english instead of pg_catalog.simple.

```sql
SHOW default_text_search_config;
```

### CODE REVIEW

**Comment by dulmandakh** [comment]
@felixxm please review and merge. Thanks

File: django/db/backends/postgresql/client.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ def runshell_db(cls, conn_params, parameters):
         host = conn_params.get('host', '')
         port = conn_params.get('port', '')
         dbname = conn_params.get('database', '')
+        options = conn_params.get('options', '')
```
Comment (felixxm) [inline django/db/backends/postgresql/client.py:18]: [`OPTIONS`](https://docs.djangoproject.com/en/3.1/ref/settings/#std:setting-OPTIONS) is a dictionary and should be consistent for all databases. I would add `'command_line_opts'` to `'OPTIONS'`, e.g.
```python
'OPTIONS': {
    'command_line_opts': 'default_text_search_config=simple',
}
```
This should be documented as a PostgreSQL-specific option in `ref/databases`, you will find similar sections for MySQL-specific options.

**Review by felixxm — COMMENTED** [review]
@dulmandakh Thanks :+1:

File: django/db/backends/postgresql/client.py (hunk 1 above)
Comment (dulmandakh) [reply django/db/backends/postgresql/client.py:18]: I expected that options for DB would be applied to dbshell too, but it didn't. per your suggestion, I have to set **default_text_search_config=simple** 2 times, one for generic and one for cli. is it correct?

Comment (felixxm) [reply django/db/backends/postgresql/client.py:18]: True, but `OPTIONS` must be a dictionary so we could use
``` 
'OPTIONS': {
    'connection_options': 'default_text_search_config=simple',
}
```
and map it to the `'options'`  in `get_connection_params()`.

Comment (dulmandakh) [reply django/db/backends/postgresql/client.py:18]: got it, I'll make the change soon. Thanks

**Comment by felixxm** [comment]
Closing due to inactivity.

**Pull request closed without merging**